lavalink/models.py

```diff
@@ -230,12 +230,12 @@ def add(self, requester: int, track: dict, extra: dict = None, index: int = None
         self.current = track
         options = {}
 
-        if 0 > end_time > track.duration or 0 > start_time > track.duration:
-            raise ValueError("end_time or start_time is either less than 0 or greater than the track's duration")
-        elif end_time == 0:
+        if end_time == 0:
             options['endTime'] = track.duration
```

Comment:
This line & the one above can be omitted, as Lavalink (by default) will play until the end of a track